scrapy/_monkeypatches.py

```diff
@@ -1,5 +1,6 @@
+import copyreg
+
 import six
```

Comment:
I believe we should be able to get rid of `six` imports all over, and of the dependency itself (at least as a direct dependency).

Reply:
Yup, I thought about that but came to the same conclusion, there are still some `six.string_types` and `six.text_type`, for instance.